### ISSUE TITLE
Organize ExperimentLog data by plugin name (Trac #61)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#65]
This will prevent naming conflicts and data for each plugin can be serialized/deserialized by the ExperimentLog class (similar to how the Protocol class works).

This will require changes to the experiment log API (again, we can use the Protocol API as a model).

Migrated from http://microfluidics.utoronto.ca/ticket/61
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "This will prevent naming conflicts and data for each plugin can be serialized/deserialized by the ExperimentLog class (similar to how the Protocol class works).\n\nThis will require changes to the experiment log API (again, we can use the Protocol API as a model).", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Organize ExperimentLog data by plugin name", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-02T00:52:51", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120302T04:31:41: ryan changed status from "new" to "accepted"

**Comment by cfobel** [comment]
Trac update at 20120306T15:01:28: 
* ryan commented:

> This ticket was fixed in changeset c914478b4506de5b55088c0a22971dfc867fe67c.


* ryan changed resolution from "" to "fixed"
* ryan changed status from "accepted" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
